### ISSUE TITLE
(possible bug) in sandbox mode, price not right

### ISSUE THREAD

**Issue by theanonymoustrader** [ccxt/ccxt#18938]
### Operating System

Windows Server 2022

### Programming Languages

Python

### CCXT Version

4.0.65

### Description

Hi everybody!
I'd like to test a strategy on Futures on the Testnet of Bybit, but I've noticed that the close price that I get is the spot price, not future.
The price are right if I set the sandbox option as False.
What I'm wrong?

### Code

```
  # Configure the symbol and timeframe
symbol = 'BTCUSDT'  # Assuming BTC perpetual contract
timeframe = '30m'

# Configure Bollinger Bands parameters
period = 20
deviation = 2

# Initialize the Bybit exchange
exchange = ccxt.bybit({
    'apiKey': api_key,
    'secret': api_secret,
    'enableRateLimit': True,
    'options': {
        'defaultType': 'future',  # testnet urls work with futures
    }

})

exchange.set_sandbox_mode(True)
exchange.options['defaultType'] = 'future' # very important set future as default type

```

**Comment by samgermain** [comment]
Could you post your full code please as a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)? I don't see anywhere where you are reading a price

**Comment by theanonymoustrader** [comment]
Sure!

`  # Configure the symbol and timeframe
symbol = 'BTCUSDT'  # Assuming BTC perpetual contract
timeframe = '30m'

# Configure Bollinger Bands parameters
period = 20
deviation = 2

# Initialize the Bybit exchange
exchange = ccxt.bybit({
    'apiKey': api_key,
    'secret': api_secret,
    'enableRateLimit': True,
    'options': {
        'defaultType': 'future',  # testnet urls work with futures
    }

})

exchange.set_sandbox_mode(True)
exchange.options['defaultType'] = 'future' # very important set future as default type

def fetch_candlestick_data():
    try:
        data = exchange.fetch_ohlcv(symbol, timeframe, limit=period)
        closes = [candle[4] for candle in data]
        return closes
    except ccxt.ExchangeError as e:
        print('Failed to fetch candlestick data:', e)
        return None

while True:
    closes = fetch_candlestick_data()
    if closes is None:
        time.sleep(5)  # Wait for 5 seconds and retry fetching data
        continue

    current_price = closes[-1]`

**Comment by carlosmiei** [comment]
Hello @theanonymoustrader,

Bybit has conflicting ids between spot and swap markets, for instance `BTCUSDT` might represent the spot market or the swap. The default type should disambiguate these situations but we have a small issue there and it is not at the moment.

I will fix this issue as soon as possible, but you can instead use the unified symbol to avoid having these issues, for instance, in this case, you could use `symbol = 'BTC/USDT:USDT' to represent the perp market and `BTC/USDT` to represent the spot market.

You can read more about it here: https://docs.ccxt.com/#/README?id=symbols-and-market-ids

**Comment by theanonymoustrader** [comment]
THank you so much for the reply,  I think that I've got another (possible) bug.
I'm using the testnet (always of bybit) and always for futures, and when I open an order the error is:
Failed to place market order t: bybit {"retCode":110007,"retMsg":"Insufficient available balance","result":{},"retExtInfo":{},"time":1692733899735}

But: I've got 18,767 k as USDT, and I'm opening an order with 0.1 as "lots".
Same result, with 0.05.
